lib/src/transformer/transformer.dart

```diff
@@ -30,23 +34,32 @@ abstract class BuilderTransformer implements Transformer, DeclaringTransformer {
     var reader = new _TransformAssetReader(transform);
     var writer = new _TransformAssetWriter(transform);
 
-    var futures = <Future>[];
-    for (var builder in builders) {
+    var allExpected = new Set<build.AssetId>();
+    await Future.forEach(builders, (builder) async {
```

Comment:
This will run each builder serially - is that the intention?

We should probably document this, one way or another.